internal/cobraext/const.go

```diff
@@ -16,4 +16,7 @@ const (
 
 	VerboseFlagName        = "verbose"
 	VerboseFlagDescription = "verbose mode"
+
+	StackVersionFlagName    = "version"
```

Comment:
I wonder if we may need a different version somewhere, so it would safer to rename this argument to `--stack-version`.